insights/combiners/modinfo.py

```diff
@@ -0,0 +1,56 @@
+"""
+ModInfo
+=======
+
+The ModInfo combiner gathers all the ModInfoEach parsers into a dictionary
+indexed by the module name.
+
+"""
+
+from insights.core.plugins import combiner
+from insights.parsers.modinfo import ModInfoEach
+
+
+@combiner(ModInfoEach)
+class ModInfo:
+    """
+    Combiner for accessing all the modinfo outputs.
+
+    Examples:
+        >>> type(modinfo_obj)
+        <class 'insights.combiners.modinfo.ModInfo'>
```

Comment:
In the test of python2, here `<type instance>` is expected, to the correct expected, better inherit the combiner class from `object` explicitly, e.g `class ModInfo(object):` Or `class ModInfo(dict):` just like the comment above.

Reply:
@xiangce do we still need to retain python2 compatibility for insights-core? I though we are py3 only here.

Reply:
@matysek : @xiangce has kindly explained on today's mtg that insights-core will be run in environments with python 2.6 and 2.7, so we need to have that compatibility. I didn't realize it before.